PluginsAndFeatures/azure-toolkit-for-intellij/rider/testData/org/cases/markup/functionapp/csharp/v4/isolatedWorker/AzureFunctionAppSyntaxHighlightingTest/testTimerTrigger_CronExpression_ValidNCrontabExpressionWithFiveFields/source/Function.cs

```diff
@@ -0,0 +1,17 @@
+using System;
+using System.Threading.Tasks;
+using Microsoft.Azure.Functions.Worker;
+using Microsoft.Extensions.Logging;
+
+namespace FunctionAppIsolated
+{
+    public static class Function
+    {
+        [Function("TimerTrigger")]
+        public static async Task RunAsync([TimerTrigger("0 */6 * * Sat,Sun")] TimerInfo myTimer, ILogger log)
```

Comment:
Update `TimerTrigger` object to 5 fields example? Please see similar tests above.

Reply:
This one has 5 fields, so should be ok?